design-library/source/_patterns/04-pages/01-one-sidebar/one-sidebar.md

```diff
@@ -0,0 +1,6 @@
+# Page: One Sidebar
+
+*Depends on*
+
+* Organism: Header
```

Comment:
We should include the other section things in here --- I should have updated before and did not